src/org/authInfo.ts

```diff
@@ -826,6 +827,15 @@ export class AuthInfo extends AsyncOptionalCreatable<AuthInfo.Options> {
         ensureString(authConfig.accessToken)
       );
 
+      const namespacePrefix = await this.getNamespacePrefix(
+        ensureString(authConfig.instanceUrl),
+        ensureString(authConfig.accessToken)
+      );
+
+      if (namespacePrefix) {
+        authConfig.namespacePrefix = namespacePrefix;
```

Comment:
`getNamespacePrefix` will return undefined if an org doesn't have a namespace prefix we just set it if there's one.